### PR TITLE
Updated RPC Flavors

### LINKED ISSUE #35: Adjust Instance Parameters
Currently the VMs are having issues. We need to up the CPU/RAM on the important VMs to ensure playbook success

### PR DESCRIPTION
**Pull request by jacobwagner**
- update rpc on rpc flavors to mirror what is actually needed
- updated infra hosts with storage network for cinder and glance apis

Fixes #35

### CODE REVIEW

**Comment by jacobwagner** [comment]
@weezer Please review

File: playbooks/roles/networking/templates/rpc-on-rpc.j2 [hunk 1]
```diff
@@ -40,7 +40,7 @@ iface br-vxlan inet static
 {% endif %}
 
 {% if storage_ip is defined %}
-{% if inventory_hostname in groups.compute %}
+{% if inventory_hostname in groups.infra or inventory_hostname in groups.compute %}
```
Comment (major) [inline playbooks/roles/networking/templates/rpc-on-rpc.j2:43]: It's possible to shorten this to:

```
{% if inventory_hostname in [groups.infra, groups.compute] %}
```

Not a big deal, but I thought it was worth mentioning. ;)

**Review by major — APPROVED** [review]
This looks okay, but I mentioned one change to improve readability.

File: playbooks/roles/networking/templates/rpc-on-rpc.j2 (hunk 1 above)
Comment (weezer) [reply playbooks/roles/networking/templates/rpc-on-rpc.j2:43]: nice catch
